WORKSPACE

```diff
@@ -382,12 +360,7 @@ http_archive(
 distdir_tar(
     name = "test_WORKSPACE_files",
     archives = [
-        "java_tools-v11.0.zip",
-        "java_tools_linux-v11.0.zip",
-        "java_tools_windows-v11.0.zip",
-        "java_tools_darwin-v11.0.zip",
         "coverage_output_generator-v2.5.zip",
-        "zulu11.37.17-ca-jdk11.0.6-linux_x64.tar.gz",
```

Comment:
Would you mind perhaps just handling java_tools_* in this PR and JDKs in other PRs? I assume JDKs have more more changes than currently in this PR, that is here only 1 of 4 JDKs is handled.

Reply:
Certainly.